### PR TITLE
add ng-ckeditor v0.2.1 with git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/esvit/ng-ckeditor
**Watch 16
  Star 239
 Fork 132**
@Amomo could u help me check this PR for #6919 ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR 
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry 
https://www.npmjs.com/package/rr-ng-ckeditor ([wait to check if it's official](https://github.com/esvit/ng-ckeditor/issues/80))
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update) 
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/ng-ckeditor/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/esvit/ng-ckeditor.git"
+  },
+  "keywords": [
+    "CKEditor",
+    "AngularJS"
+  ],
+  "author": "Vitalii Savchuk",
+  "license": "BSD 3-clause",
```
Comment (PeterDaveHello) [inline ajax/libs/ng-ckeditor/package.json]: Wrong.

File: ajax/libs/ng-ckeditor/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/esvit/ng-ckeditor.git",
+    "basePath": "",
+    "files": [
+      "ng-ckeditor*.+(js|css|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/ng-ckeditor/package.json]: Why not just `ng-ckeditor*`?

Comment (maruilian11) [inline ajax/libs/ng-ckeditor/package.json]: i think both are ok, we used to use that.

**Comment by maruilian11** [comment]
@Amomo i revised the `license` and modify `filemap` to `ng-ckeditor*`, could you help me check this PR again? thanks!

**Pull request merged**